pages/[type]/[id]/moderation.vue

```diff
@@ -70,9 +70,9 @@
     <section id="messages" class="universal-card">
       <h2>Messages</h2>
       <p>
-        This is a private conversation thread with the Modrinth moderators. They will message you
-        for issues concerning your project on Modrinth, and you are welcome to message them about
-        things concerning your project.
+        This is a private channel with the Modrinth moderators. They may contact you about your
+        project concerns, and you can also reach out with any questions or issues related to your
+        Modrinth project.
```

Comment:
Why was this changed? The old text was perfectly fine, and the new text is confusing (what does "they may contact you about your project concerns" mean?)